src/app/store/Models/AlbumsModel.ts

```diff
@@ -6,24 +6,42 @@ import uuid from "react-native-uuid"
 export interface Album {
   id: Readonly<string>
   name: string
-  artworkIds?: string[]
-  documentIds?: string[]
+  artworkIds: string[]
```

Comment:
```suggestion
  artworkIds: ArtworkId[]
```
since we have these, lets do that?